plone/namedfile/scaling.py

```diff
@@ -387,6 +415,30 @@ def scale(
         )
         if info is None:
             return  # 404
+
+        try:
+            (orig_width, orig_height) = self.getImageSize(fieldname)
+        except:
```

Comment:
You can get an Unauthorized error when getting the original image object.

We changed this a bit in a065e173ad8b725403e5ee3a31904ea47f614494 but that caused Unauthorized errors in tests (plone.app.caching, plone.formwidget.namedfile, plone.outputfilters). I can't reproduce this Unauthorized error in real life.

In 0f4f960f475f6cdbcc653799bfce9acb41e84ed5 I changed it to specifically catch the Unauthorized error.